applications/SubsCrypt.md

```diff
@@ -0,0 +1,185 @@
+
+* An overview of the technology stack to be used  
+As we explained, we want to provide a service that will fit into different stacks with minimum effort so that we will implement our side of this service with the following stack:
+- User/developer Dashboard (Front) => Vue.js and RPC API to contract
+- Smart Contract => Rust and Ink on edgeware
+- UI integration components => ES6 and purejs library
+- Backend high-level API to connect to smart contract => Django, node.js for this milestone
+
+### Ecosystem Fit 
+As we have been searching for similar ideas in the blockchain ecosystem, we found EIP1337 that has a similar idea, but the implementation is low level, and it does not help either developers or users have a smooth subscription platform.
+
+## Team :busts_in_silhouette:
+
+### Team members
+* Name of team leader
+    Saber Zafarpoor (Product manager & Blockchain Designer)
+    Hadi Esna (CTO & Blockchain Designer)
+* Names of team members	
+    Ahmad Salimi(Software Engineer)
+    Alireza Ilami(Data Analyst)
+    Sepehr Kianian(Front-End developer & Blockchain developer)
+    Amir Sadra Abdolahi(Front-End developer & Blockchain developer)
+    Yasaman Shad(Graphic Designer & UX)
+    ...(we will hire blockchain researchers and UI developers to enhance the quality)
+
+
+### Contact
+* **Contact Name:** Saber Zafarpoor
+* **Contact Email:** szafarpoor@ce.sharif.edu
+* https://oxydev.ir
+
+
+### Team's experience
+**OxyDev** consists of young enthusiast students/developers who graduated or undergraduates at #1 university of Iran (Sharif University of Technology) trying to make web3 easier to use and integrate into real-world services. We have developed several Defi projects in Tron and Ethereum Ecosystem and expanded our team to make venturous leaps to play our roles in making a more user-friendly world in the next generation of Web(Web3). We have done several projects of web2 that make our team aware of the pros and cons of web2. 
+
+### Team Code Repos
+* https://github.com/oxydev
+* Our previous projects can not be publicly announced(NDA)
+
+### Team LinkedIn Profiles
+* https://www.linkedin.com/in/szafarpoor/
+* https://www.linkedin.com/in/ahmadsalimi/
+* https://www.linkedin.com/in/hadiesnaashari/
+* https://www.linkedin.com/in/alirezailami/
+
+## Development Roadmap :nut_and_bolt: 
+We will divide our project into four milestones. For the first milestone, we will implement our smart contract can be deployed to a substrate chain using ink! Which is a smart contract substrate pallet. The second and third milestones will be implemented in parallel. The last milestone will be connecting the components to work seamlessly together.
```

Comment:
Have you done any dev work in ink! yet?

Reply:
We are currently learning it.

Reply:
We are trying to implement [this contract](https://github.com/oxydev/SubsCrypt-solidity/blob/main/SubsCrypt.sol) in ink! .